app/views/account/workspaces/create.js.erb

```diff
@@ -1,6 +1,7 @@
 <% if @workspace.persisted? %>
-  $('.navbar-workspaces').html(`<%= j render 'account/workspaces/show_workspaces', workspaces: @workspaces %>`);
-  $('.modal').removeClass('is-active');
+  $('.navbar-workspaces').html('<%= j render 'account/workspaces/show_workspaces', workspaces: @workspaces %>');
+  $('.modal .modal-form').html('<%= j render 'account/workspaces/modal_flash' %>');
+  setTimeout(function(){ $('.modal').removeClass('is-active') }, 1000);
```

Comment:
bootstrap modal?
if yes - use bootstrap JS event to close modal

Reply:
no its bulma